Documentation/platforms/risc-v/esp32c3/index.rst

```diff
@@ -291,7 +291,7 @@ Bluetooth    No
 CAN/TWAI     Yes
 CDC Console  Yes    Rev.3
 DMA          Yes
-eFuse        No
+eFuse        Yes    Virtual mode supported
```

Comment:
I think the Virtual mode is like a "dry run" right? So I suggest using "(also Virtual mode supported)", otherwise it could be interpreted as only Virtual mode is supported

Reply:
Yes, Virtual mode is simulation of efuse unit to check.